output/output.go

```diff
@@ -37,9 +43,22 @@ func New(rep requester.Report, cfg config.Global) *Output {
 			Config:     cfg,
 			FinishedAt: time.Now(),
 		},
+
+		log: func(v ...interface{}) {
+			outputLogger.Println(ansi.Bold(fmt.Sprint(v...)))
```

Comment:
Nitpick: I would not force bold formatting on everything this logger will print, but let the caller choose when they want it.